src/pip/_internal/commands/search.py

```diff
@@ -7,16 +7,14 @@
 
 from pip._vendor import pkg_resources
 from pip._vendor.packaging.version import parse as parse_version
-# NOTE: XMLRPC Client is not annotated in typeshed as on 2017-07-17, which is
```

Comment:
You seem to have moved the comment from search.py, instead of download.py.

Reply:
yeah, yo right 🤦🏽‍♂️. Let me change that ASAP